packages/core/tests/rspack-provider/plugins/__snapshots__/default.test.ts.snap

```diff
@@ -1277,7 +1275,9 @@ exports[`applyDefaultPlugins > should apply default plugins correctly when prod
         "_options": {
           "compress": "{"passes":1,"pure_funcs":[],"drop_console":false}",
           "exclude": undefined,
-          "extractComments": "true",
+          "extractComments": {
+            "condition": "@preserve|@lic|@cc_on|^/**!",
```

Comment:
true will be resolve to "@preserve|@lic|@cc_on|^/**!", so the minify result should be the same